src/ansys/workbench/core/workbench_client.py

```diff
@@ -381,6 +381,28 @@ def download_file(self, file_name, show_progress=True, target_dir=None):
             pbar.close()
         return file_name
 
+    def download_project_archive(self, archive_name, show_progress=True):
+        """Create and download the project archive.
+
+        Parameters
+        ----------
+        archive_name : str
+            The name of the project archive to use, without the file extension.
```

Comment:
```suggestion
            Name of the project archive to use, without the file extension.
```
Ansys style is to omit the leading article in the brief description so that it always starts with an important word.